backend/judging-algorithm/parser.ts

```diff
@@ -38,3 +39,12 @@ export function parseRoomsCSV (roomCsvFilePath: string): Room[] {
 export function parseRotationTimeCSV (rotationTimesCsvFilePath: string): RotationTime[] {
   return parseCsv<RotationTime>(rotationTimesCsvFilePath, true)
 }
+
+async function updateAllData() {
+  await updateJudgeData();
+  await updateRoomsData();
+  await updateRotationTimes();
+  await updatesTeamData();
+}
+
+updateAllData();
```

Comment:
does this call need to go here? should we just export the function and call it elsewhere?

Reply:
Sure! Should we call the function in app.ts then?